src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -49,82 +37,83 @@ describe('addAggregatorAndDestinationToMemo', () => {
       finalAssetAddress: FINAL_ASSET_ADDRESS,
       minAmountOut,
       slippageBps,
-      finalAssetPrecision: 18,
-      chainNamespace: CHAIN_NAMESPACE.Evm,
+      longtailTokens: DEFAULT_THORCHAIN_ASSETS_LIST,
     })
 
     expect(modifiedMemo).toBe(
-      `=${MEMO_PART_DELIMITER}e${MEMO_PART_DELIMITER}${RECEIVE_ADDRESS}${MEMO_PART_DELIMITER}${expectedL1AmountOut}${MEMO_PART_DELIMITER}ss${MEMO_PART_DELIMITER}${affiliateBps}${MEMO_PART_DELIMITER}${AGGREGATOR_TWO_LAST_CHARS}${MEMO_PART_DELIMITER}${FINAL_ASSET_ADDRESS}${MEMO_PART_DELIMITER}206301621786412`,
+      `=${MEMO_PART_DELIMITER}e${MEMO_PART_DELIMITER}${RECEIVE_ADDRESS}${MEMO_PART_DELIMITER}${expectedL1AmountOut}${MEMO_PART_DELIMITER}ss${MEMO_PART_DELIMITER}${affiliateBps}${MEMO_PART_DELIMITER}${AGGREGATOR_TWO_LAST_CHARS}${MEMO_PART_DELIMITER}${SHORTENED_FINAL_ASSET_ADDRESS}${MEMO_PART_DELIMITER}941367142801`,
```

Comment:
If we're not concerned about this guy doing the shortening logic and it is tested separately, do we want to keep things cleaner and easier to read by not interpolating, and having the assertions use a regular string instead?

Sounds like having interpolation would be the concern of `makeMemoWithShortenedFinalAssetAmount`, not this guy which is only really concerned about bytes size FWIW

Reply:
Removed most of the string interpolation, I kept the very big address so it's more readable

Reply:
Much better now, tyvm!